install/connections.csv

```diff
@@ -86,48 +86,48 @@ CARD,10,Cairn Hill,Portable,,2010-09-04T23:23:02Z,2010-09-29T00:00:00Z
 CASA,10,Glenthorne,Short Period,,1994-06-19T20:22:00Z,1994-08-06T11:15:00Z
 CASM,10,Castle River,Short Period,,1993-10-18T05:51:00Z,1994-03-11T20:39:00Z
 CAW,10,Cannon Point,,,2003-05-20T02:00:00Z,9999-01-01T00:00:00Z
-CAW,11,Cannon Point,,,2003-05-20T02:00:00Z,2003-05-20T03:00:00Z
+CAW,11,Victoria University,EARSS Network,,2003-05-20T02:00:00Z,2003-05-20T03:00:00Z
```

Comment:
Shall this be 
`CAW,11,Cannon Point,EARSS Network,,2003-05-20T02:00:00Z,2003-05-20T03:00:00Z`
instead?

also, I think the line above (CAW location code 10) needs also to be changed 
either to 
`CAW,10,Victoria University,EARSS Network,,2003-05-20T02:00:00Z,9999-01-01T00:00:00Z`
or to 
`CAW,10,Cannon Point,EARSS Network,,2003-05-20T02:00:00Z,9999-01-01T00:00:00Z`

Reply:
The digitser was at Victoria University and the sensor at Cannon Point.
(at the same time the local digitiser 10 was being used, this would have been a transition from using 10 -> 11, but keeping 10 for the local Q330).